scripts/perf/start_database/start_database.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -euo pipefail
+
+source scripts/perf/helper.inc.sh
+
+start_cloud_sql
```

Comment:
I guess seeing the  flow of the jenkins jobs would make this obvious, but I'm not sure how this is different from the other database-using jobs that these need to start the database manually but they don't?

Reply:
This system currently isn't cloning a database since the tests are read-only. There just one database that gets reused for all runs:

1. Start database
2. Deploy GCE instance
3. Run tests
4. Delete GCE instance
5. Stop database